eudemo/config/build_config.json

```diff
@@ -123,7 +123,7 @@
         }
     },
     "PF coils": {
-        "run_mode": "read",
+        "run_mode": "run",
```

Comment:
If we're changing this to run can we remove the json files or is it still useful to have them stored?

Reply:
I think it is still useful, but yeah it is annoying with all the diffs. Let me try and get to the point where all the saved files are consistent with each other, and then we can gitignore them, so that one can always use "read" and still get something that makes sense.